### ISSUE TITLE
Provide a Tutorial which keeps the Simplicity Promise

### ISSUE THREAD

**Issue by ghost** [meteor/tutorials#101]
I tried to get started with Meteor, using the Tutorials.

I expected a super-simple get started. Instead, I find instructions like "meteor remove blaze-html-templates", which suggests to remove files before adding the Angular related ones.

I don't think that this fulfills the promise you make "Build a simple app in minutes"?

### Expected

I would expect something as simple as:

```shell
# Creates an angular application based on the web-template
$ meteor create simple-todos --angular --template=web

# Adds an Entity to the Model
$ meteor create entity Todo title:string text:text
```

...and so on, with minimal effort and thinking, thus newcomers get results immediately (and bored core-developers, which refuse to plumb around).

Is this actually achievable with the current status of Meteor?

**Comment by abernix** [comment]
Thanks for raising this concern.

There are absolutely other ways to start an Angular project aside from using the steps outlined in this repository, however Angular is not the default view layer included in Meteor, thus the additional steps and modifications to the "minutes" timeline you're referring to.  There used to be an option to run `meteor create --example angular2-boilerplate` but in an effort to keep things more actively maintained, that example is now maintained in a [separate repository](https://github.com/bsliran/angular2-meteor-base).  It can still be `git clone`'d, and you'll be given the exact command instruction if you run the `--example` command I included above.

That being said, you can absolutely still get a simple app built in minutes, however Angular is a relatively complicated application from the get-go so keep that in mind. :smile:

We're definitely open to improving this, but a boilerplate is a great way to start when using a view layer that isn't the default that Meteor offers. (e.g. Blaze).

**Comment by ghost** [comment]
There are several things we will most possibly agree:

* A core developer (of a product) cannot assess what is easy for a newcomer
* A core developer cannot produce entry-level documentation (he has far too much knowledge)
* A newcomer does not know many things
* An high-experienced developer does not always *want* details, just to get started (despite he having the brain/experience to grasp them).
* Documentation should be simple, and the ideal documentation is the non-existent one (because the product functionality is near-perfect and self-explaining).

I understand your struggle to present 3 different UI tools, and to produce and maintain 3 tutorials in parallel. Thats a small hell. But there are solutions, like e.g.

a) Focus on **one** UI Layer, or

b) Separate the sections (create the Server first, like you are LoopBack, then excel with the 3 different UI tutorials)

### Goal

Do you agree that this here would be a nice goal?

 "With Meteor, you can create simple websites, database-driven websites, API Servers (UI less), API's with Admin UI's. Whatever your project type, getting started is really a matter of minutes, and you can be live within an hour. Meteor allows you to focus on what really matters for you: your design, your domain-model, your api."

### First Step

I wanted to showcase you "simple documentation", but then i hit on a barrier (see #100)

So, the issue #100 must be verified/solved first, and then the documentation-contribution process itself. In an ideal setup, the user which filed issue #102, would just have clicked "Edit this file", and send a pull-request in.

Lets fix all this, because I have the feeling that Meteor is far simpler as it is presented. So, a decent getting-started will help to increase Meteors adoption even more.

**Comment by abernix** [comment]
We've fixed #100 and #102.

> I would expect something as simple as:
> _(two command snipped)_

The tutorial would no longer be a tutorial if we just had you run a single (or two) commands.  Additionally, that's not how Meteor works so we can't change the tutorial to do something Meteor doesn't so.  If you're suggesting Meteor be something else or work differently, then that's an issue for another place (Meteor itself).

> There are several things we will most possibly agree:

I actually don't agree with most of your bullet-points, but most of them are not relevant to the tutorial so I'm not going to address it.

> a) Focus on one UI Layer, or
> b) Separate the sections (create the Server first, like you are LoopBack, then excel with the 3 different UI tutorials)

Unfortunately, that's not a realistic process for building an app in Meteor in a tutorial walk-through since we'd be confusingly switching your view layer half-way through.
